app/src/main/java/com/kickstarter/ui/activities/DeepLinkActivity.kt

```diff
@@ -72,6 +76,19 @@ class DeepLinkActivity : BaseActivity<DeepLinkViewModel.ViewModel?>() {
             .compose(bindToLifecycle())
             .compose(Transformers.observeForUI())
             .subscribe { finish() }
+
+        viewModel.outputs.startPreLaunchProjectActivity()
+            .compose(bindToLifecycle())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe {
+                val intent = Intent().getPreLaunchProjectActivity(
```

Comment:
This piece could be moved to `ActivityExt.kt` as example `Activity.startRootCommentsActivity` extension function